spec/datetimeformat.html

```diff
@@ -63,6 +63,11 @@ <h1>Abstract Operations For DateTimeFormat Objects</h1>
           <td>`"timeZoneName"`</td>
           <td>`"short"`, `"long"`</td>
         </tr>
+        <tr>
```

Comment:
No idea if we discussed this already, but do we have any preference to keep these entries in logical order, so that [[FractionalSecondDigits]] is directly preceded by [[Second]]? (Technically it's observable how these entries are ordered, but it shouldn't make much of a difference in practice.)